apisix/admin/init.lua

```diff
@@ -148,6 +149,9 @@ local function run()
             core.response.exit(400, {error_msg = "stream mode is disabled, " ..
                                "can not add stream routes"})
         end
+        core.config.init()
+        local router_stream = require("apisix.stream.router.ip_port")
+        router_stream.stream_init_worker(filter)
```

Comment:
here: https://github.com/apache/apisix/pull/8219#discussion_r1024937749 means move `router_stream.stream_init_worker(filter)` to `function _M.init_worker()`, not `local function run()`.

Reply:
done